homeassistant/components/lcn/climate.py

```diff
@@ -110,12 +107,9 @@ def supported_features(self) -> int:
         return const.SUPPORT_TARGET_TEMPERATURE
 
     @property
-    def temperature_unit(self) -> UnitTemperatureT:
+    def temperature_unit(self) -> str:
         """Return the unit of measurement."""
-        # Config schema only allows for: TEMP_CELSIUS and TEMP_FAHRENHEIT
-        if self.unit == pypck.lcn_defs.VarUnit.FAHRENHEIT:
```

Comment:
I think we should keep this logic. Optionally can be addressed in a separate PR.